tests/lib.rs

```diff
@@ -193,16 +193,20 @@ fn cleanup_file(f: &str) {
     }
 }
 
-fn control_fdr(test: &str, event_str: &str, alpha: f64, local: bool) {
+fn control_fdr(test: &str, events: Vec<&str>, alpha: f64, local: bool) {
```

Comment:
```suggestion
fn control_fdr(test: &str, events: &[&str], alpha: f64, local: bool) {
```

Take a slice here (this enables to either use Vec or Array below. Best practice: always make function params as generic as possible.